src/reachability/utils/reachability.jl

```diff
@@ -161,3 +218,36 @@ end
 
 # Define overapproximate for SymbolicIntervalMasks and SymbolicIntervalGradients
 LazySets.overapproximate(set::_SymIntOrGrad, ::Type{Hyperrectangle}) = Hyperrectangle(low=low(set), high=high(set))
+function temp_overapprox(set::Star; optimizer=GLPK.Optimizer)
```

Comment:
temp_overapprox and issubset will be removed once the LazySets Star set PR finishes.